### ISSUE TITLE
[$250] Screen Reader: Settings-Profile: The currently active menu button is not announced as such

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#75556]
## Action Performed
Prerequisites:
- The user is signed in

1. Using JAWS + Chrome, open the site https://new.expensify.com/settings/profile
2. Press Tab key to focus the currently selected menu button 'Profile'
3. Note how the element is displayed and announced.

## Expected Result
The visually selected menu button should be announced as selected.

## Actual Result
As the user navigates through the Account profile buttons, all the elements are only announced with the name of the button, and no context as to whether or not the button has already been selected or not.
- 'Profile, button'

HTML:
<div aria-label="Profile" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-18u37iz r-1wtj0ep r-13qz1uu r-1awozwy r-1xfd6ze r-102gzdx r-11f147o r-3pj75a" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style="background-color: rgb(230, 225, 218);"><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1awozwy r-1777fci r-iyfy8q"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#03D47C"><circle cx="10" cy="7" r="3"></circle><path d="M10 1c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9m5.9 12.7C14.6 12.1 12.4 11 10 11s-4.6 1.1-5.9 2.7C3.4 12.6 3 11.4 3 10c0-3.9 3.1-7 7-7s7 3.1 7 7c0 1.4-.4 2.6-1.1 3.7"></path></svg></div></div><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div class="css-g5y9jx r-18u37iz r-1awozwy r-dnmrzs"><div dir="auto" data-selection-scrapper-hidden-element="false" class="css-146c3p1 r-1udh08x r-1udbk01 r-1iln25a" style="color: rgb(0, 46, 34); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; flex-shrink: 1; margin-left: 12px; white-space: pre; direction: ltr; max-width: 100%; line-height: 20px;">Profile</div></div></div></div></div><div class="css-g5y9jx r-18u37iz" style="min-height: 40px;"></div></div></div></div>

## Area issue was found
Settings–Profile

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Use the aria-current to announce the current menu button as such.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1917" height="1002" alt="Image" src="https://github.com/user-attachments/assets/0f5cdde1-c7dc-4aea-b97f-3d8b4f432400" />

https://github.com/user-attachments/assets/a20291b4-ba0c-478c-9ddb-08fb1638f7eb

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6985987
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @Burhan-Rashid</details>
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022030981639635489104</li>
        <li>Upwork Job ID: 2030981639635489104</li>
        <li>Last Price Increase: 2026-03-09</li>
<li>Automatic offers: </li>
<ul>
<li>aimane-chnaif | Contributor | 110741981</li>
</ul></li>
    </ul>
</details>

**Comment by rushatgabhane** [comment]
@MelvinBot fix this issue please. suggest a solution

**Comment by rushatgabhane** [comment]
reassigned because of urgency https://expensify.slack.com/archives/C02NK2DQWUX/p1773261967885959

**Comment by Burhan-Rashid** [comment]
This is only deployed to staging currently. We still need to wait for production deployment and regression period of 7 days before payment and closing of the issue, Right? @mallenexpensify

**Comment by mallenexpensify** [comment]
ooops, got ahead of myself.  Thx @Burhan-Rashid.  You've yet to be paid, @aimane-chnaif has been.  Please tag me once payment is due, cuz automation's been failing on a most Accessibility issues.  Thx
